### PR TITLE
Issue#542 fix

### PR DESCRIPTION
**Pull request by we-ai**
This PR is to fix issue #542. The followings are details:
- #542 (https://github.com/episphere/connect/issues/542) fix.
- Cleaned up some codes for clarity.

### CODE REVIEW

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
See few comments, we can discuss if necessary. Besides that looks great. Love that we have meaningful variable names now.

File: src/fieldToConceptIdMapping.js [hunk 1]
```diff
@@ -64,8 +65,31 @@ export default {
     discardFlag: 762124027,
     deviationNotFound: 982885431,
 
+    // biospecimen collection
+    collection:{
+        id:820476880,
+        idIsManuallyEntered: 387108065,
+        accountIdEntered: 148996099,
```
[the remark this answers is not in the document]
Comment (we-ai) [reply src/fieldToConceptIdMapping.js:72]: removed 148996099

File: src/fieldToConceptIdMapping.js [hunk 2]
```diff
@@ -64,8 +65,31 @@ export default {
     discardFlag: 762124027,
     deviationNotFound: 982885431,
 
+    // biospecimen collection
+    collection:{
+        id:820476880,
+        idIsManuallyEntered: 387108065,
```
[the remark this answers is not in the document]
Comment (we-ai) [reply src/fieldToConceptIdMapping.js:71]: removed 387108065

File: src/shared.js [hunk 3]
```diff
@@ -819,7 +805,79 @@ export const removeBag = async(boxId, bags) => {
     return await response.json();
     
 }
+export const specimenCollection = {
```
[the remark this answers is not in the document]
Comment (Gbarra9) [reply src/shared.js:808]: In my opinion, I like the constant "specimenCollection" for reference and readability in the getInstituteSpecimensList function found in the events.js file.

https://github.com/episphere/biospecimen/pull/415/files?diff=unified&w=0#diff-aa3209e98840e6d236562086e3c6942a28f0c6beef6897d036367d09cc313c59R691-R710

**Review by Gbarra9 — APPROVED** [review]

File: src/shared.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (anthonypetersen) [reply src/shared.js:808]: > In my opinion, I like the constant "specimenCollection" for reference and readability in the getInstituteSpecimensList function found in the events.js file.
> 
> https://github.com/episphere/biospecimen/pull/415/files?diff=unified&w=0#diff-aa3209e98840e6d236562086e3c6942a28f0c6beef6897d036367d09cc313c59R691-R710

I don't disagree about the readability, my only concern is that if / when we have to add new tubes it's just another place that we have to update constants.

**Review by anthonypetersen — APPROVED** [review]

File: src/shared.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (we-ai) [reply src/shared.js]: Moved the constant to tubeValidation.js for easier maintenance.

**Review by abhinavjonnada82 — APPROVED** [review]

**Pull request merged**